pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -499,14 +506,84 @@ class Build {
                 context.println "ERROR: ${configurePath} was not found. Exiting..."
                 throw new Exception()
             }
+
+            // Get JVM Version
+            String jvmMajorPath = "workspace/target/metadata/jvm_version/major.txt"
+            String jvmMinorPath = "workspace/target/metadata/jvm_version/minor.txt"
+            String jvmSecurityPath = "workspace/target/metadata/jvm_version/security.txt"
+            String jvmTagsPath = "workspace/target/metadata/jvm_version/tags.txt"
+
+            context.println "INFO: Attempting to read workspace/target/metadata/jvm_version/*..."
+
+            try {
```

Comment:
Not sure if it's worth extracting each of these into their own small functions

Reply:
Have added a `println` for each file read, any increase in clarity is good.